api/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/stretchr/testify v1.7.0
 	google.golang.org/genproto v0.0.0-20211118181313-81c1377c94b1
-	google.golang.org/grpc v1.41.0
+	google.golang.org/grpc v1.47.0
```

Comment:
The title mentions 1.48, but actually 1.47

Reply:
:) I guess it's a typo in the PR title as the latest stable release is 1.47. lgmt.

Reply:
Good catch. Reworded.